packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -243,30 +245,28 @@ export default function MainFeedLayout({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [sortingEnabled, selectedAlgo, loadedSettings, loadedAlgo]);
 
-  const isMobile = useViewSize(ViewSize.MobileL);
-  const isLaptop = useViewSize(ViewSize.Laptop);
-
-  const getImage = () => {
-    if (isMobile) {
-      return cloudinary.feed.bg.mobile;
+  const getPadding = () => {
+    if (isV1Search) {
+      return '!pt-2';
     }
 
-    return isLaptop ? cloudinary.feed.bg.laptop : cloudinary.feed.bg.tablet;
+    return shouldUseFeedLayoutV1 && '!pt-0';
   };
 
   const FeedPageComponent = shouldUseFeedLayoutV1 ? FeedPageLayoutV1 : FeedPage;
 
   return (
-    <FeedPageComponent className="relative">
-      {searchVersion === SearchExperiment.V1 && !isFinder && (
-        <img
-          className="absolute left-0 top-0 w-full max-w-[58.75rem]"
-          src={getImage()}
-          alt="Gradient background"
+    <FeedPageComponent className={classNames('relative', getPadding())}>
+      {isV1Search && !shouldUseFeedLayoutV1 && !isFinder && <FeedGradientBg />}
+      {isSearchOn && search}
+      {feedProps && (
+        <Feed
+          {...feedProps}
+          className={
+            shouldUseFeedLayoutV1 && !isFinder && 'laptop:!max-w-[36.5rem]'
```

Comment:
I believe the content should be maxed out, we're just missing paddings on desktop screens. I think I'd rather introduce paddings then another set size width.

Reply:
This is an experiment to introduce a new layout that might be better overall, part of introducing the improvements can be the size of the width itself. I think we are accurate in specifying the exact values rather than always adjusting how much should be the padding.

Reply:
But in case we stay with this, we have already introduced a max width for the higher up container - 680px and aligned that with the width on profile pages as well, in case we keep this and want to change this max size of the higher up container later on to let's say 700px - then we would need to change this size as well here to something bigger, rather then having a 24px padding left and right, in which case we wouldn't have to do anything.

Reply:
That actually makes sense. Let's go with that.